.gitignore

```diff
@@ -38,3 +38,6 @@ hack/tools/bin
 
 # vscode
 .vscode/
+
+# macOS
+.DS_Store
```

Comment:
should you also add the binary name `fulcio-certificate-maker` here so it doesn't get checked in by accident?

Reply:
good point / thanks.